src/components/Orders/OldOrder.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+
+function OldOrder(props) {
+    const { date, totalAmount, products, menu, order, receiverDeletedOrder } = props;
+
+    const reorderHandler = () => {
+        const date = new Date();
```

Comment:
It would be great to move date formatting function to an external module. Let me know if you need help with this